### DISCUSSION TITLE
테라폼 관련 질문 드립니다.

### DISCUSSION THREAD

**Discussion by b100to** [tedilabs/fastcampus-devops#74] [Q&A] [answered]
변수를 config.yaml에서 관리하게 되면 시크릿 변수들은 어떻게 사용하시나요?

**Comment by posquit0** [comment, marked as answer]
기밀 정보는 하드코딩하거나 버전 관리에 포함되지 않도록 하는 것을 원칙으로 접근합니다.
팀에서 사용하는 솔루션에 따라 AWS Secrets Manager, SSM Parameter Store, HashiCorp Vault 등과 연동시킬 수 있습니다.
